api/v1alpha1/hostedcluster_types.go

```diff
@@ -142,6 +145,29 @@ type Release struct {
 	Image string `json:"image"`
 }
 
+// TODO maybe we have profiles for scaling behaviors
+type ClusterAutoscaling struct {
+	// Maximum number of nodes in all node groups.
+	// Cluster autoscaler will not grow the cluster beyond this number.
+	// +kubebuilder:validation:Minimum=0
+	MaxNodesTotal *int32 `json:"maxNodesTotal,omitempty"`
+
+	// Gives pods graceful termination time before scaling down
+	// default: 600 seconds
```

Comment:
can we do any :validation here?

Reply:
not sure i follow, could you explain a little more please?

Reply:
Is there a value for this fields that would make the cluster autoscaler fail? If so can we structural schema validate that here so the bad input does not make it through?

Reply:
ah! thanks for clarifying that. i'll have to check, but i think the only invalid values would be negative numbers. i will double check though.